canonical/webteam.canonical.com
# Update teams list

### Done
Update teams list

### QA steps
1. go to https://webteam-canonical-com-137.demos.haus/team/random
2. make sure all teams are listed in a random order
